if __name__ == "__main__":

    class prueba(Exception):

        def __init__(self,error):
            super().__init__()
            self.error = error



        def __str__(self) -> str:
            return self.error





    try:

        raise prueba('esto es un error')


    except prueba as err:

        print(err)    
